src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="Memory\AsReadOnlyMemory.cs" />
```

Comment:
nit: Can you move this down to the ItemGroup with the rest of the Memory tests? i.e. here: https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/System.Memory.Tests.csproj#L80